index.js

```diff
@@ -46,7 +46,8 @@ class PostCSSCompiler {
 		this.map = this.config.map ?
 			Object.assign({}, defaultMapper, this.config.map) :
 			defaultMapper;
-		this.getDependencies = progeny({rootPath, reverseArgs: true});
+      const progenyOpts = Object.assign({}, {rootPath, reverseArgs: true}, cfg.progeny);
```

Comment:
This can be replaced with `Object.assign({rootPath, reverseArgs: true}, cfg.progeny)`, right?